__tests__/index.tests.js

```diff
@@ -39,3 +40,63 @@ describe('error 404', ()=>{
         })
     })
    
+    describe('GET /api', () => {
+      test('GET:200 sends an object with the endpoints to the client', () => {
+        return request(app)
+          .get('/api')
+          .expect(200)
+          .then((response) => {
+            expect(response.body.endpoints).toEqual(expect.any(Object));
+            expect(Object.keys(response.body.endpoints)).toEqual(
+              expect.arrayContaining(["GET /api", "GET /api/topics","GET /api/articles",  ]))
+              
+                     
+          });
+      });
+
+    }
+    )
+    describe('error 404 on GET /api bad path', ()=>{
+      test('GET:404', ()=>{
+          
+              return request(app)
+              .get('/apii')
+              .expect(404)
+     
+          })
+      })
+      describe('GET /api/articles/:article_id', () => {
+        test('GET:200 sends an object with the article corresponding to the id given to the client', () => {
+          
+          return request(app)
+            .get(`/api/articles/1`)
+            .expect(200)
+            .then((response) => {
+              console.log(response.body)
+              expect(response.body.article).toEqual(expect.any(Object));
+              expect(Object.keys(response.body.article[0])).toEqual(
+                expect.arrayContaining(["author",
+                "title",
+                "article_id",
+                "body",
+                "topic",
+                "created_at",
+                "votes",
+                "article_img_url"  ]))
+                expect(response.body.article[0].article_id).toBe(1)
+              
+                       
+            });
+          }
+        )
+        test('GET:404 on bad path', ()=>{
+          
+          return request(app)
+          .get('/api/articless/1')
```

Comment:
This is testing the 404 error handler you already tested. We need to test a different 404 here which is caused by no records being found e.g. if our article id is 5000